### PR TITLE
[chore] Prepare release v1.0.0-rcv0013/v0.80.0

### PR DESCRIPTION
**Pull request by opentelemetrybot**
The following commands were run to prepare this release:
- make chlog-update VERSION=v1.0.0-rcv0013/v0.80.0
- make prepare-release GH=none PREVIOUS_VERSION=1.0.0-rcv0012 RELEASE_CANDIDATE=1.0.0-rcv0013 MODSET=stable
- make prepare-release GH=none PREVIOUS_VERSION=0.79.0 RELEASE_CANDIDATE=0.80.0 MODSET=beta

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -284,6 +284,8 @@ check-contrib:
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/config/configtls=$(CURDIR)/config/configtls"
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/config/internal=$(CURDIR)/config/internal"
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/confmap=$(CURDIR)/confmap"
+	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/connector=$(CURDIR)/connector"
+	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/connector/forwardconnector=$(CURDIR)/connector/forwardconnector"
```
Comment (codeboten) [inline Makefile:288]: Hmmm... this looks incorrect

Comment (Aneurysm9) [reply Makefile:288]: I'm not sure why this change is appearing here.  It's already been made on `main` and was brought into this branch by merging main into the source branch of this PR via #7944.

Comment (codeboten) [reply Makefile:288]: Possibly because of the squash merge? It might be easier to close this pr and trigger a new one :(

Comment (Aneurysm9) [reply Makefile:288]: I'm not sure that's necessary, but I can do it if you'd prefer.  It looks like this will still end up with the same end-state.

**Review by codeboten — APPROVED** [review]

**Pull request merged**